contrib/gitian-build.sh

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2016 The Bitcoin Core developers
+# Copyright (c) 2021 The dogecoin Core developers
```

Comment:
```suggestion
# Copyright (c) 2016 The Bitcoin Core developers
# Copyright (c) 2021 The Dogecoin Core developers
```
Copyrights are additive, rather than replacing.